ci-operator/config/coreos/coreos-assembler/coreos-coreos-assembler-rhcos-4.14.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.fedoraproject.org/fedora:37
```

Comment:
```suggestion
      FROM registry.fedoraproject.org/fedora:38
```
?

Reply:
Wondering if we missed other instances of that for 4.13

Reply:
4.13 is on F37: https://github.com/coreos/coreos-assembler/blob/rhcos-4.13/Dockerfile

So it's only here